boids.cabal

```diff
@@ -17,7 +17,7 @@ library
   build-depends:       base
                      , gloss
                      , random
-  exposed-modules:     Boids
+  exposed-modules:     Boids, Utils
```

Comment:
This is usually written in a column, one module per line (just like `build-depends` above.)